code/API_definitions/click-to-dial.yaml

```diff
@@ -2,17 +2,41 @@ openapi: 3.0.0
 
 info:
   title: Click to Dial Service API
-  version: wip
+  version: 0.1.0
```

Comment:
This change should have been done only within the (pre-)release PR.
And for the prepared release the correct version is 0.1.0-alpha.1